### PR TITLE
pythonPackages.swagger-ui-py: init at 0.3.0

### PR DESCRIPTION
**Pull request by MajesticMullet**
###### Motivation for this change

Adding swagger-ui-py package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/swagger-ui-py/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  propagatedBuildInputs = [
+    jinja2
+    pyyaml
+  ];
+
+  meta = with lib; {
+      description = "Swagger UI for Python web framework, such as Tornado, Flask, Quart, aiohttp, Sanic and Falcon.";
+      hompage = "https://github.com/PWZER/swagger-ui-py";
+      license = with licenses; [mit];
+      maintainers = with maintainers; [PWZER];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/swagger-ui-py/default.nix:28]:
```suggestion
      license = licenses.mit;
      maintainers = with maintainers; [ PWZER ];
```

File: pkgs/development/python-modules/swagger-ui-py/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    sha256 = "1910zvkp2613s4ysb61k2gkaw3m1hdg2433kbqil80mqm915i1ix";
+  };
+
+  propagatedBuildInputs = [
+    jinja2
+    pyyaml
+  ];
+
+  meta = with lib; {
+      description = "Swagger UI for Python web framework, such as Tornado, Flask, Quart, aiohttp, Sanic and Falcon.";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/swagger-ui-py/default.nix:25]: Descriptions normally shouldn't start with the package name or end with a dot.

Comment (MajesticMullet) [reply pkgs/development/python-modules/swagger-ui-py/default.nix]: Changed and noted.

File: pkgs/development/python-modules/swagger-ui-py/default.nix (hunk 1 above)
Comment (MajesticMullet) [reply pkgs/development/python-modules/swagger-ui-py/default.nix]: Updated

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102996` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python27Packages.swagger-ui-py</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
@MajesticMullet please resolve the merge conflict.

**Comment by MajesticMullet** [comment]
Sorry it took so long - Conflicts should now be resolved. @SuperSandro2000

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 102996` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.swagger-ui-py</li>
<li>python38Packages.swagger-ui-py</li>
<li>python39Packages.swagger-ui-py</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

python37Packages.swagger-ui-py:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>
python38Packages.swagger-ui-py:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>
python39Packages.swagger-ui-py:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 102996` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.swagger-ui-py</li>
<li>python38Packages.swagger-ui-py</li>
<li>python39Packages.swagger-ui-py</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

python37Packages.swagger-ui-py:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>
python38Packages.swagger-ui-py:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>
python39Packages.swagger-ui-py:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>

**Pull request closed without merging**